commands/trigger.go

```diff
@@ -547,15 +640,24 @@ func (t *Trigger) Update(Client *whisk.Client, args []string) error {
 	retTrigger, httpResp, err := Client.Triggers.Get(qualifiedName.GetEntityName())
 
 	if err != nil && httpResp.StatusCode == 404 {
-		t.Create(Client, args)
+		if createErr := t.Create(Client, args); createErr != nil {
```

Comment:
This is new. If trigger get fails then trigger update now fails, correct?

Reply:
Correct. 
1. If trigger get fails with 404, then we create the trigger. If creating the triggers fails, we have to report back to the user. Previously we do not report any error when we create trigger in update and that was a bug. 
2. If trigger fails with other reason, execution will fall into next else if, and we report the error message as well.